systemtest/kibana.go

```diff
@@ -314,5 +314,27 @@ func CreateSourceMap(t testing.TB, sourcemap []byte, serviceName, serviceVersion
 		Value: id,
 	})
 
+	t.Cleanup(func() {
+		DeleteSourceMap(t, result.ID)
+	})
+
 	return result.ID
 }
+
+// DeleteSourceMap deletes a source map with the given ID.
```

Comment:
FWIW I think this is cleaner anyway. We upload source maps through Kibana, so we should delete them through Kibana too; otherwise our system tests may be affected by internal Kibana changes.